src/clj/chocolate/processing.clj

```diff
@@ -0,0 +1,49 @@
+(ns chocolate.processing
+  (:require [chocolate.protobuf.interface :as pb-if]
+            [chocolate.routes.websockets :as ws]
+
+            ; just to be sure they are compiled
+            [chocolate.protobuf.person]
+            [chocolate.protobuf.message]))
+
```

Comment:
So we'll get a compilation error if these have not been gen'd using protoc?

Reply:
Both the protoc AND compiling the java files into classes. This “should” happen automatically on a “lein run”